index.html

```diff
@@ -0,0 +1,450 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars" id="fa-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav">
+        <div><i class="fa fa-times" id="closeIcon"></i></div>
+        <!-- <div class="nav-links" id="navLinks"> -->
+        <ul >
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
```

Comment:
this link is displaying the about me section as expected and congrats for that but in the other hand it doesn't close the mobile menu, this issue is happening because you forgot to create a click event listener for it